src/translations/en.json

```diff
@@ -1808,7 +1808,7 @@
         },
         "integrations": {
           "caption": "Integrations",
-          "description": "Manage integrations",
+          "description": "Manage devices & services",
```

Comment:
You don't really manage devices here, since that happens in the actual "Devices" section. So that should at least not be so prominent here in the description to not mislead and rather focus on the "integration" part.

Perhaps: "Manage integrations to services, devices, ...".

Reply:
Done.